docs/en_US/SearchSpaceSpec.md

```diff
@@ -26,9 +26,19 @@ Take the first line as an example. `dropout_rate` is defined as a variable whose
 All types of sampling strategies and their parameter are listed here:
 
 * {"_type":"choice","_value":options}
-  * Which means the variable value is one of the options, which should be a list. The elements of options can themselves be [nested] stochastic expressions. In this case, the stochastic choices that only appear in some of the options become conditional parameters.
+
+  * Which means the variable value is one of the options, which should be a list. The elements of options can themselves be **nested** stochastic expressions. In this case, the stochastic choices that only appear in some of the options become conditional parameters.
+
+  * An simple [example](../../examples/trials/mnist-cascading-search-space/search_space.json) of [nested] search space definition. The elements of options in the list must be dictionary with one key `_name` and its value pair. Here is a [sample]((../../examples/trials/mnist-cascading-search-space/sample.json) ) which users can get from nni with **nested** search space definition. Tuners which support nested search space is as follows:
+
+    - Random Search 
+    - TPE
+
+    - Anneal
+    - Evolution
```

Comment:
Which means the variable's value is one of the options. Here 'options' should be a list. Each element of options is a number of string. It could also be a **nested** sub-search-space (), this sub-search-space takes effect only when the corresponding element is chosen. The variables in this sub-search-space could be seen as conditional variables.

An simple [example](../../examples/trials/mnist-cascading-search-space/search_space.json) of [nested] search space definition. If an element in the `options` list is a dict, it is a sub-search-space, and for our built-in tuners you have to add a key '_name' in this dict, which helps you to identify which element is chosen. Accordingly, here is a [sample]((../../examples/trials/mnist-cascading-search-space/sample.json) ) which users can get from nni with **nested** search space definition. Tuners which support nested search space is as follows:

Reply:
@leelaylay

Reply:
Thank you.